arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -2266,7 +2266,7 @@
 				3E6AE3F31C6D1C920026E59C /* Find route */,
 				3EC39C191C9A5770003F6459 /* Route around barriers */,
 				3E6AE3EA1C6D10F70026E59C /* Offline routing */,
-				3ED75D001ED39EDF009B1F75 /* Find service area interactive */,
+				3ED75D001ED39EDF009B1F75 /* Find service area (interactive) */,
```

Comment:
Renaming a sample is more than just updating a README. Something like this that affects the project file itself should really go in through a PR to `v.next`. Changing the name in the README and the plist is fine, but please revert this group name change and instead submit a separate PR for it to `v.next`.